changelogs/fragments/6289-bugfix-flatpak-check-if-already-installed.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - flatpak - fixes idempotency detection issues. In some cases the module could fail to properly detect already existing flatpaks because of a parameter witch only checks the installed apps. (https://github.com/ansible-collections/community.general/pull/6289).
```

Comment:
```suggestion
  - flatpak - fixes idempotency detection issues. In some cases the module could fail to properly detect already existing Flatpaks because of a parameter witch only checks the installed apps (https://github.com/ansible-collections/community.general/pull/6289).
```